x/authenticator/ante/pubkey.go

```diff
@@ -1,9 +1,7 @@
 package ante
 
-// TODO: This is just a fork of the sdk's SetPubKeyDecorator. Here I'm allowing any pubkey so I can test the ante.
```

Comment:
might still be worth it to have a comment explaining the diffs between this one and the default. Useful for future devs

Reply:
updated the comment explaining